src/components/autocomplete/js/autocompleteController.js

```diff
@@ -203,7 +207,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
       $mdUtil.enableScrolling();
     }
 
-    angular.element($window).off('resize', positionDropdown);
+    angular.element($window).off('resize', onWindowResize);
```

Comment:
We need to have the same fn-Object to be able to de-register the listener properly. 

> Also having as a normal function is more clean...